src/main/flight/gps_rescue.h

```diff
@@ -25,7 +25,9 @@ typedef struct gpsRescue_s {
     uint16_t angle; //degrees
     uint16_t initialAltitudeM; //meters
     uint16_t descentDistanceM; //meters
-    uint16_t rescueGroundspeed; // centimeters per second
+    uint16_t targetLandingAltitudeCm; //centimeters
+    uint16_t targetLandingDistanceCm; //centimeters
+    uint16_t rescueGroundspeed; //centimeters per second
```

Comment:
Additions to the structure must go at the end otherwise the parameter group will be corrupted.

Reply:
ok